lib/public/views/Runs/RunPerDataPass/RunsPerDataPassOverviewPage.js

```diff
@@ -47,11 +47,16 @@ export const RunsPerDataPassOverviewPage = ({ runs: { perDataPassOverviewModel }
 
     return h('', [
         h('.flex-row.justify-between.items-center', [
-            dataPass.name ? h('h2', `Good, physics runs of ${dataPass.name}`) : null,
+            dataPass.match({
+                Success: (payload) => h('h2', `Good, physics runs of ${payload.name}`),
+                Failure: () => h('h2.danger', 'Failed to fetch Data Pass information'),
+                Loading: () => h('h2', 'Good, physics runs of ...'),
+                NotAsked: () => h('h2', 'Good, physics runs of ...'),
```

Comment:
Not asked you can put `Good physics runs` and loading you may put a spinner?

Reply:
added spinner